net/sched/sch_skbprio.c

```diff
@@ -200,7 +199,7 @@ static int skbprio_init(struct Qdisc *sch, struct nlattr *opt,
 	q->highest_prio = 0;
 	q->lowest_prio = SKBPRIO_MAX_PRIORITY - 1;
 	if (!opt) {
-		sch->limit = max(qdisc_dev(sch)->tx_queue_len, min_limit);
+		sch->limit = 64;
```

Comment:
Style: move `sch->limit = 64;` before the `if` statement. This will eliminate the need for `{` and `}` in the `if` statement and the code will make obvious that `sch->limit` is always initialized.